experimental/plugins.md

```diff
@@ -31,9 +31,14 @@ The following plugins exist:
   driver plugin which provides a virtual, multi-host network for containers.
 
 * The [Calico plugin](https://github.com/metaswitch/calico-docker) is a network
-  driver plugin which provides a multi-host network for containers with routes 
+  driver plugin which provides a multi-host network for containers with routes
   distributed by BGP.
 
+* The [REX-Ray plugin](https://github.com/emccode/rexraycli) is a volume plugin
+  which is written in Go and provides advanced storage functionality for many
+  platforms including EC2, OpenStack, XtremIO, and ScaleIO.
```

Comment:
I don't see anything in the documentation of that repo that suggests that this is in fact a Docker volume plugin - ie, something that listens on a socket and responds to the Docker volume API

@calavera perhaps you can help me out?